packages/core/src/lib/utils.mts

```diff
@@ -16,3 +16,20 @@ export const breakLines = (content: string, width: number): string =>
         .map((str) => str.trimEnd()),
     )
     .join('\n');
+
+/**
+ * Creates a 0-based index out of an integer, wrapping around if necessary.
+ * @param {number} max The maximum count
+ * @param {number} value The value to convert to index
+ */
+export const index = (max: number, value: number) => ((value % max) + max) % max;
```

Comment:
I'd prefer not exposing an extra (and small) helper from `@inquirer/core`. Let's inline and duplicate the utility in both prompts.